pkg/util/scrubber/scrubber.go

```diff
@@ -71,6 +84,10 @@ var blankRegex = regexp.MustCompile(`^\s*$`)
 type Scrubber struct {
 	singleLineReplacers []Replacer
 	multiLineReplacers  []Replacer
+
+	// CondFunction is a function that can be used to conditionally apply a replacer.
+	// If the function returns false, the replacer will not be applied.
+	CondFunction func(repl Replacer) bool
```

Comment:
While I get you, especially with the second sentence, `CondFonction` on its own is a bit unclear, WDYT of something like "ShouldApply" or similar?

Reply:
I named it shouldApply first, I can comeback to this name for sure 👍🏻